charts/vela-core/crds/standard.oam.dev_routes.yaml

```diff
@@ -116,7 +116,6 @@ spec:
                   issuerName:
                     type: string
                   type:
-                    default: Issuer
```

Comment:
we should delete the legacy crd dir, this crd will not install in target k8s 1.15

Reply:
How about the others?

legacy/charts/vela-core-legacy/crds/standard.oam.dev_routes.yaml
legacy/charts/vela-core-legacy/crds/standard.oam.dev_rollouttraits.yaml
legacy/charts/vela-core-legacy/crds/standard.oam.dev_rollouts.yaml
legacy/charts/vela-core-legacy/crds/standard.oam.dev_autoscalers.yaml
legacy/charts/vela-core-legacy/crds/standard.oam.dev_metricstraits.yaml